index 7.js

```diff
@@ -0,0 +1,135 @@
+"use strict"
+
+const items = [
```

Comment:
Better to add `id` to the products as well.

Reply:
Sure, I'll handle it.